app/src/main/java/com/nextcloud/talk/conversationlist/ConversationsListActivity.kt

```diff
@@ -1469,5 +1509,7 @@ class ConversationsListActivity :
         const val CLIENT_UPGRADE_GPLAY_LINK = "https://play.google.com/store/apps/details?id="
         const val HTTP_SERVICE_UNAVAILABLE = 503
         const val MAINTENANCE_MODE_HEADER_KEY = "X-Nextcloud-Maintenance-Mode"
+        const val MENTION: String = "mention"
```

Comment:
same constants and values in 2 classes, so I would suggest to put them in a separate constants class to remove the duplication.